onedal/common/_policy.py

```diff
@@ -48,12 +48,8 @@ def __init__(self):
 
 
 if _is_dpc_backend:
-    from onedal._device_offload import DummySyclQueue
```

Comment:
This is the only place where `DummySyclQueue` is used. I am ok with this refactoring, but it worth also remove it definition as well. I didn't find the description for these changes, make sense to move into separate branch or add into commit messages on the merge this work as well, to be clear other dev/users why and when this were removed.